infrastructure/public/common/bucket-and-user.cdk/package-lock.json

```diff
@@ -25,115 +25,323 @@
             }
         },
         "node_modules/@aws-cdk/aws-events": {
-            "version": "1.15.0",
-            "resolved": "https://registry.npmjs.org/@aws-cdk/aws-events/-/aws-events-1.15.0.tgz",
-            "integrity": "sha512-kkPn4PdgniSMED/1YTimVhAw5KIZfwbHS1CFtxPxAHtHVN8vQ0ewHgVlfZGaP9O96FpWkW+kuyc40Hfz8LES9g==",
-            "dependencies": {
-                "@aws-cdk/aws-iam": "1.15.0",
-                "@aws-cdk/core": "1.15.0"
+            "version": "1.204.0",
+            "resolved": "https://registry.npmjs.org/@aws-cdk/aws-events/-/aws-events-1.204.0.tgz",
+            "integrity": "sha512-KnfUmtv+4RhydD9+5CHFxNJxtgn7+Xftwfwg1G7qV/tWYPFHcNIvhlSOgwDrQPa+pTo1MmkiUN0lAR0G8B/cbw==",
+            "deprecated": "AWS CDK v1 has reached End-of-Support on 2023-06-01.\nThis package is no longer being updated, and users should migrate to AWS CDK v2.\n\nFor more information on how to migrate, see https://docs.aws.amazon.com/cdk/v2/guide/migrating-v2.html",
```

Comment:
Rather than merge this, we probably want to get this updated to AWS CDK v2 or remove the functionality.